chia/full_node/full_node_api.py

```diff
@@ -1273,13 +1273,32 @@ async def respond_compact_proof_of_time(self, request: timelord_protocol.Respond
     @execute_task
     @peer_required
     @api_request
-    async def new_compact_vdf(self, request: full_node_protocol.NewCompactVDF, peer: ws.WSChiaConnection):
+    @bytes_required
+    async def new_compact_vdf(
+        self, request: full_node_protocol.NewCompactVDF, peer: ws.WSChiaConnection, request_bytes: bytes = b""
+    ):
         if self.full_node.sync_store.get_sync_mode():
             return None
+
+        if len(self.full_node.compact_vdf_sem._waiters) > 20:
+            self.log.debug(f"Ignoring NewCompactVDF: {request}, _waiters")
+            return
+
+        name = std_hash(request_bytes)
```

Comment:
Can we just use `request.header_hash` here and avoid this sha256 entirely?

Reply:
No, one header can have multiple VDFs that can be blueboxed.